src/index.js

```diff
@@ -3,9 +3,30 @@
  */
 import store, { getMessageInfo, changeTimeFormat } from './store';
 // example of generating HTML with js
-//
-function renderThreads() {
-  const inbox = store.mailboxes.INBOX;
+
+function addMailboxList() {
+  const mailboxes = Object.keys(store.mailboxes);
+  return mailboxes.map(mailbox => `
+    <li class="nav-item">
+      <button class="mailbox-item active" type="button" id="${mailbox}" >${mailbox}</button>
```

Comment:
A nice thing to do here might be to consider creating a function to format your mailbox names. Some come prepended with "category_" so you could get rid of this with a fairly simple function :)